openslides_backend/action/motion/reset_recommendation.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.schema import schema_version
+from ..generics import UpdateAction
+from ..register import register_action
+
+reset_schema = {
+    "$schema": schema_version,
+    "title": "Reset motion recommendation schema",
+    "description": "An array of objects with motion id",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {"id": {"type": "integer"}},
```

Comment:
But you should at least use `Motion().get_properties("id")` here so that we don't double any field definitions

Reply:
Update: see other comment.

Reply:
Okay